### PR TITLE
External Redirect: Concrete Implementation

### PR DESCRIPTION
**Pull request by hedvigoscar**
Bit hard to test currently unfortunately, but we'll get there soon enough!

### CODE REVIEW

File: src/pages/Embark/index.tsx [hunk 1]
```diff
@@ -196,6 +197,11 @@ export const EmbarkRoot: React.FunctionComponent<EmbarkRootProps> = (props) => {
     <StorageContainer>
       {(storageState) => (
         <EmbarkProvider
+          externalRedirects={{
+            Offer: () => {
+              history.push('/new-member/offer')
```
Comment (sampettersson) [inline src/pages/Embark/index.tsx]: use a constant for this maybe, so that we don't have to change it in two places :)

Comment (palmenhq) [reply src/pages/Embark/index.tsx]: just checking does this work with `/en/new-member` also?

Comment (hedvigoscar) [reply src/pages/Embark/index.tsx]: I don't know 😞 

I'll make sure to try it, there are some things I need to do before I can check this one off.

I'll mark it as do not merge!

**Comment by hedvigoscar** [comment]
Mostly obsolete at this point

**Pull request closed without merging**